install/kubernetes/helm/istio-init/files/crd-10.yaml

```diff
@@ -1004,6 +1028,8 @@ spec:
     kind: RbacConfig
     plural: rbacconfigs
     singular: rbacconfig
+    shortNames:
+    - rbaccfg
```

Comment:
too complicated.

Reply:
rcfg?

Reply:
FYI: This is deprecated by `ClusterRbacConfig` in 1.1 and will be removed probably in 1.2 or 1.3.

Reply:
Prefer to not add this